ecs-cli/modules/clients/aws/cloudformation/client.go

```diff
@@ -124,7 +124,7 @@ func newClient(config *config.CommandConfig, client cloudformationiface.CloudFor
 }
 
 // CreateStack creates the cloudformation stack by invoking the sdk's CreateStack API and returns the stack id.
-func (c *cloudformationClient) CreateStack(template, stackName string, capabilityIAM bool, params *CfnStackParams) (string, error) {
+func (c *cloudformationClient) CreateStack(template, stackName string, capabilityIAM bool, params *CfnStackParams, tags []*cloudformation.Tag) (string, error) {
```

Comment:
What do you think about (perhaps in a future PR) refactoring this func to take in a `cloudformation.CreateStackInput` struct + capabilityIAM? With the addition of `tags` we're now passing in 4 values that are its propertied which we're directly setting.

Reply:
...it would also clean up a lot of the mock vars we're passing around in `/cluster_app_test`

Reply:
That's a good idea. I'd prefer to do that in a future PR, but also happy to do it in this.